public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -74,9 +73,16 @@ export const RulesGroup = React.memo(({ group, namespace, expandAll, viewMode }:
   const isListView = viewMode === 'list';
   const isGroupView = viewMode === 'grouped';
 
-  const deleteGroup = () => {
-    dispatch(deleteRulesGroupAction(namespace, group));
-    setIsDeletingGroup(false);
+  const deleteGroup = async () => {
+    const namespaceName = decodeGrafanaNamespace(namespace).name;
+    const groupName = group.name;
+    const dataSourceName = rulesSourceToDataSourceName(namespace.rulesSource);
+
+    const ruleGroupIdentifier: RuleGroupIdentifier = { namespaceName, groupName, dataSourceName };
+    try {
+      await deleteRuleGroup.execute(ruleGroupIdentifier);
+      setIsDeletingGroup(false);
+    } catch (_err) {}
```

Comment:
I'm not sure why I disabled this but it seems dangerous, have to inspect this 🕵️

Reply:
Looks suspicious